### DISCUSSION TITLE
[Project Beta]: Apply board filters to items created on the board.

### DISCUSSION THREAD

**Discussion by despairblue** [community/community#7529] [Projects and Issues] [answered]
If a board is filtered and you create an item there you always get this:

![The item was created but it has been hidden because it does not match the current filter.](https://user-images.githubusercontent.com/927609/141078202-138122b7-7da9-4cb5-a859-9fa99df78712.png)

Instead the current filters should automatically be applied to the new ticket. That's how notion and jira work.

Right now I always have to go to a table view to do that. When I forget I have to go to the table view and fix it there.

**Comment by markoudev** [comment, marked as answer]
Seems to be already worked on! https://github.com/github/feedback/discussions/6469

**Reply by despairblue** [reply]
Thanks. I searched before, but couldn't find anything. 💜
